web/frontend/src/components/slam-2d-render.vue

```diff
@@ -161,16 +288,23 @@ const updateCloud = (pointcloud: Uint8Array) => {
     }
   }
 
-  scene.add(points);
-  scene.add(marker);
-  scene.add(intersectionPlane);
-};
+  // construct grids
+  const axesHelper1 = createAxisHelper('AxesPos', Math.PI / 2)
```

Comment:
1. Why do we need to add anything other than the `point`s when the pointcloud changes? 
2. Can't we add all the other entities only once on mount?
3. Could we please name these axis helpers after what they are for? As an example, if they are for the positive & negative axies can we call them `axesPos` and `axesNeg` as you do their names?

Reply:
you need `points` for the gridhelper method since the grids need to be spaced at 1m

Reply:
otherwise, axesHelper1 and axesHelper2 (can be moved into the mount function and be added from there.

Reply:
moving `axesHelper1` and `axesHelper2` (with more descriptive variable names) to onMount makes sense to me.